diff --git a/node_modules/@react-pdf/unicode-properties/dist/unicode-properties.esm.js b/node_modules/@react-pdf/unicode-properties/dist/unicode-properties.es.js
similarity index 97%
rename from node_modules/@react-pdf/unicode-properties/dist/unicode-properties.esm.js
rename to node_modules/@react-pdf/unicode-properties/dist/unicode-properties.es.js
index 1224171..b4ab787 100644
--- a/node_modules/@react-pdf/unicode-properties/dist/unicode-properties.esm.js
+++ b/node_modules/@react-pdf/unicode-properties/dist/unicode-properties.es.js
@@ -1,6 +1,6 @@
-var UnicodeTrie = require('unicode-trie');
-var data = require('./data.json');
-var trieBuffer = require('./trie.json');
+import UnicodeTrie from 'unicode-trie'
+import data from './data.json'
+import trieBuffer from './trie.json'
 
 // Trie is serialized as a Buffer in node, but here
 // we may be running in a browser so we make an Uint8Array
