diff --git a/node_modules/@react-pdf/renderer/index.d.ts b/node_modules/@react-pdf/renderer/index.d.ts
index 5acf756..c58332c 100644
--- a/node_modules/@react-pdf/renderer/index.d.ts
+++ b/node_modules/@react-pdf/renderer/index.d.ts
@@ -512,6 +512,7 @@ declare module '@react-pdf/renderer' {
   const PDFRenderer: typeof ReactPDF.PDFRenderer;
   const version: typeof ReactPDF.version;
   const pdf: typeof ReactPDF.pdf;
+  const renderToStream: typeof ReactPDF.renderToStream;
   const PDFViewer: typeof ReactPDF.PDFViewer;
   const BlobProvider: typeof ReactPDF.BlobProvider;
   const PDFDownloadLink: typeof ReactPDF.PDFDownloadLink;
@@ -532,6 +533,7 @@ declare module '@react-pdf/renderer' {
     PDFRenderer,
     version,
     pdf,
+    renderToStream,
     PDFViewer,
     BlobProvider,
     PDFDownloadLink,
